src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -36,6 +36,8 @@ internal class ItemManager : IAsyncCompletionItemManager
         private readonly RecentItemsManager _recentItemsManager;
         private readonly IGlobalOptionService _globalOptions;
 
+        public const string AggressiveDefaultsMatchingOptionName = "AggressiveDefaultsMatchingOption";
```

Comment:
what does 'aggressive defaults' mean?

Reply:
From @dpugh:
> With aggressive matching turned on, the default was use as long as what the user typed in the applicable to span was a prefix of the item

I believe the purpose of this option is to provide a way to experiment with different selection logic.

Reply:
>  the default was use as long as what the user typed in the applicable to span was a prefix of the item

i still don't know what this means.  can we have a comment explaining this somewhere (ideally with examples).